### PR TITLE
fix: check internet with the outside world

### PR DESCRIPTION
**Pull request by PradoGuilherme**

### CODE REVIEW

File: bin/command/command_install.js [hunk 1]
```diff
@@ -59,86 +60,90 @@ module.exports = {
...
-          console.log(' EXIST: '.bgBlue, 'The component ', componentName, ' has already been installed!')
-          console.log('If you need reinstall put --update in command.'.grey)
-          callback()
-          return false
-        }
-        component.install(componentJson, function (err) {
-          if (err) {
-            console.log(' ERROR: '.bgRed, 'Sorry, the component could not be installed at this time.\n', err)
+  install: async function (componentName, callback) {
+    if (await isOnline()) {
```
Comment (fabiorogeriosj) [inline bin/command/command_install.js]: MobileUI is compatible with old nodejs version (without async/await) please use callback method

File: bin/command/command_install.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ var getfont = require('../utils/getfont')
 var template = require('../utils/template')
 var environment = require('../utils/environment')
 var request = require('request')
+const isOnline = require('is-online')
```
Comment (fabiorogeriosj) [inline bin/command/command_install.js]: use `var` for old nodejs versions

Comment (PradoGuilherme) [reply bin/command/command_install.js]: Updated to `var`

File: bin/command/command_install.js (hunk 1 above)
Comment (PradoGuilherme) [reply bin/command/command_install.js]: Sorry about this mistake. I already updated the code to use native module `dns` from `node` to check internet connection.

**Pull request merged**